.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 2.10.0
+current_version = 2.12.0
```

Comment:
The latest published version is actually `2.11.1`, but we had a kerfuffle with git history.